administrator/components/com_media/views/media/tmpl/default.php

```diff
@@ -11,6 +11,30 @@
 $doc    = JFactory::getDocument();
 $params = JComponentHelper::getParams('com_media');
 
+/**
+ * Obtain list of supported providers
+ *
+ * @return string
+ *
+ * @since __DEPLOY_VERSION__
+ */
+function getProviderInfo()
```

Comment:
This needs to be done in the view, the template files should not have functions.